src/containers/bookform.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable react/no-array-index-key */
+import React from 'react';
+
+class BooksForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+      category: '',
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSelect = this.handleSelect.bind(this);
+  }
+
+  handleChange({ target }) {
+    this.setState({ title: target.value });
+  }
+
+  handleSelect({ target }) {
+    this.setState({ category: target.value });
+  }
+
+  render() {
+    const CATEGORIES = [
+      'Action',
+      'Biography',
+      'History',
+      'Horror',
+      'Kids',
+      'Learning',
+      'Sci-Fi',
+    ];
+
+    const { title, category } = this.state;
+    return (
+      <form>
+        <input type="text" name="title" value={title} onChange={this.handleChange} />
+        <select name="category" onChange={this.handleSelect} value={category}>
+          {CATEGORIES.map((cat, i) => (
+            <option key={i} value={cat}>
```

Comment:
So keys should not be indexes, this warning should have been given by React, it is bad practice, it messes up the DOM if something is removed from the collection of object